dot/rpc/modules/childstate_integration_test.go

```diff
@@ -0,0 +1,7 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package modules
```

Comment:
What's the purpose of that file?

Reply:
This is the childstate's integration test file, I will update it in another PR, this change is just to fix the license (same for other integration tests files)